### ISSUE TITLE
socks5 proxy support

### ISSUE THREAD

**Issue by maxfunky** [fireice-uk/xmr-stak#1676]
does xmr-stak have support socks5 proxy in config ?

in a way similar to cpuminer-multi `-x SOCKS5://127.0.0.1:9050`

**Comment by Spudz76** [comment]
No, however you may be able to use something like [this](https://github.com/tonyseek/rsocks) where you configure a localhost port to give to xmr-stak as a "normal TCP endpoint", and configure your socks5 endpoint in rsocks.

Uses Python so it "should" work on pretty much any OS... and there could be others I only looked for a minute.  Keywords "socks5 reverse proxy" mostly worked.

**Comment by maxfunky** [comment]
but would it be hard to implement directly into xmr-stak ?

**Comment by Rock1111** [comment]
ssh -N -f -L  3333:pool.supportxmr.com:3333    root@remoteserver_can_connect_to_pool

then try localhost:3333
